articles/openshift/oshift-scaling-references.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Considerations When Scaling an OpenShift Cluster | UKCloud Ltd
+description: Gives information on things to consider when scaling a cluster
+services: openshift
+author: Kieran O'Neill
+toc_rootlink: References
+toc_sub1: 
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Considerations When Scaling an OpenShift Cluster
```

Comment:
Same change to capitalisation:
Considerations when scaling an OpenShift cluster